pkg/syncthing/syncthing.go

```diff
@@ -702,32 +668,34 @@ func (s *Syncthing) Restart(ctx context.Context) error {
 }
 
 // Stop halts the background process and cleans up.
-func (s *Syncthing) Stop(force bool) error {
-	pidPath := filepath.Join(s.Home, syncthingPidFile)
-	pid, err := getPID(pidPath)
-	if os.IsNotExist(err) {
-		return nil
-	}
-
-	if !force {
-		if pid != s.pid {
-			log.Infof("syncthing pid-%d wasn't created by this command, skipping", pid)
-			return nil
-		}
-	}
-
-	if err := s.cleanupDaemon(pid, force); err != nil {
+func (s *Syncthing) Stop(wait bool) error {
+	pList, err := process.Processes()
```

Comment:
in my experience, listing processes and waiting on them is a bit flaky and time consuming, do you think we should only do this if the pidPath is not valid?

Reply:
can you also validate if process.Processes() works when you're not an admin on the machine? (it's something common on windows environments)

Reply:
It is slower. I think I am going to do it just at the beginning of the `up` sequence and in parallel with the rest of the `up` sequence.
For windows, yes, it also works for non-admin users